apps/web/src/pages/get-started/components/GetStartedAnimation.tsx

```diff
@@ -11,15 +13,29 @@ interface IGetStartedAnimationProps {
 // uses `public` as the default base directory
 const ROOT_ANIMATION_PATH = `animations/get-started`;
 const STATE_MACHINE_NAME = 'SM';
-const STATE_MACHINE_INPUT_NAME = 'white theme';
+const STATE_MACHINE_INPUT_NAME = 'theme';
+const LIGHT_THEME_INPUT = 2;
+const DARK_THEME_INPUT = 0;
```

Comment:
Can we add a comment about these values. why it needs to be 0 and 2 and not 0, 1?

Reply:
left a comment